src/System.Private.ServiceModel/tests/Common/Infrastructure/src/project.json

```diff
@@ -6,8 +6,8 @@
     "System.Net.Http": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-23921",
-    "System.ServiceModel.Duplex": "4.0.1-rc3-23921",
-    "System.ServiceModel.Http": "4.0.11-rc3-23921",
+    "System.ServiceModel.Duplex": "4.1.0-rc3-23921",
```

Comment:
Eric, the error I'm getting is...
1>EXEC : error : System.Linq.Expressions 4.0.0 provides a compile-time reference assembly for System.Linq.Expressions on DNXCore,Version=v5.0, but there is no run-time assembly compatible with win7-x64. [C:\OSS\wcf\build.proj]

I updated every project that had an explicit dependency to it to use version "4.0.11-rc3-23921", I still get the failure, and then I saw that xunit 2.1.0 depends on the 4.0.0 version transitively, so I'm wondering if I need to add an explicit reference to S.Linq.Expressions in all our test projects to get around this issue?